### DISCUSSION TITLE
Problems getting old look back after update

### DISCUSSION THREAD

**Discussion by Lockna** [davatorium/rofi#1700] [Theming]
Hey guys, I updated rofi and after the update it looked pretty weird. Currently I have the problem that I still have a border and the elements in the listview are differently formated. The odd positions are the way I want it to look, the even positions have a white background and I don't know why.
![after](https://user-images.githubusercontent.com/43321748/188496660-5a221c86-ef7f-419b-afb5-141965ebe628.png)
![before](https://user-images.githubusercontent.com/43321748/188496800-ea5a43cf-9c8f-4a76-b96c-75b130bb603e.png)
The first picture is after the updated and the second before.
My configuration looks like this: https://pastebin.com/7GE50TGc
Any ideas?

**Comment by DaveDavenport** [comment]
from what version to what version.

That looks like you made a significant jump. You might want to check out the sticky issue.
https://github.com/davatorium/rofi/issues/1397

**Comment by Lockna** [comment]
Well I know that I moved to version 1.7.5, from which i don't really know.
I tried that in the sticky issue, but nothing worked

**Reply by DaveDavenport** [reply]
The theme as pasted above works fine for me in the latest release. 
![rofi-2022-09-05-2112-00000](https://user-images.githubusercontent.com/406245/188503936-2d7b0d44-2f9e-43be-81be-96b6d507d490.png)

Try moving the theme into a 'mytheme.rasi' file and add:
```css
@theme "mytheme"
```

in your configuration instead of your theme.
How it is configured now you are extending the default theme, not loading a new one.

**Reply by Lockna** [reply]
Yeah, thank you very much. This fixed the problem
